starter-code/package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "node ./bin/www",
+    "start-dev": "nodemon ./bin/www"
   },
   "dependencies": {
```

Comment:
missing two packages needed to complete the project:
- `express-session` - middleware to manage browser sessions
- `connect-mongo` - extension to store those sessions in our mongo database